demucs/separate.py

```diff
@@ -42,9 +42,9 @@ def load_track(track, audio_channels, samplerate):
 
     if wav is None:
         print(f"Could not load file {track}. "
-              "Maybe it is not a supported file format? ")
+              "Maybe it is not a supported file format? ") if args.verbose != 0 else None
```

Comment:
thanks for proposing a fix, however I think your PR would be cleaner if you implemented a 
`print_error(*args, verbose=args.verbose)` and a `print_info(*args, verbose=args.verbose)`  rather than the `if else` construct.